testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js

```diff
@@ -269,7 +270,7 @@ QUnit.module('popup integration', {
         });
 
         const instance = $dropDownButton.dxDropDownButton('instance');
-        const $popupContent = $(getPopup(instance).content());
+        const $popupContent = $(getPopup(instance)._$content);
```

Comment:
Is it a necessary change?

Reply:
Yes, content() returns popup content, but we need overlay content